### PR TITLE
Tweak remove/hide track confirm message to match confirm exit message

### PR DESCRIPTION
**Pull request by ninomp**
This is seeking to fix https://bugs.launchpad.net/mixxx/+bug/1947992.

This is a follow-up of #4330.

### CODE REVIEW

File: src/widget/wtracktableview.cpp [hunk 1]
```diff
@@ -814,7 +816,11 @@ void WTrackTableView::hideOrRemoveSelectedTracks() {
             return;
         }
 
-        response = QMessageBox::question(this, tr("Confirm track removal"), message);
+        response = QMessageBox::question(this,
```
Comment (ronso0) [inline src/widget/wtracktableview.cpp:819]: how would that part be reached given the return; in the else branch above?

Comment (ninomp) [reply src/widget/wtracktableview.cpp:819]: The `return;` above is for the case when the track model does not support hiding nor removing tracks (e.g. in Missing tracks or Hidden tracks).

Comment (ronso0) [reply src/widget/wtracktableview.cpp:819]: uhaa, I lost track in the condesnsed mobile view. foregt evrything I said :\

**Comment by foss-** [comment]
Testing 2.4-alpha-1034-g91bcb96ec6 (HEAD) from https://github.com/mixxxdj/mixxx/actions/runs/1608937872 does not resolve the UI issue reported in https://bugs.launchpad.net/mixxx/+bug/1947992:

**Comment by ronso0** [comment]
@foss- Is the button appearance of the (hide message) identical for all skins?
  I'm asking because with this PR the only difference between the "Close Mixxx?" and the "Confirm Track Hide" message is the _parent_ widget: "Close Mixxx?" child of mixxxmainwindow, "Confirm Track Hide" child of WTrackTableView. The latter may have a stylesheet applied, with snippets like `WLibrary QPushButton ...`.

**Comment by foss-** [comment]
This is more divers than anticipated.

Deere (all caps):
Screenshot 2021-12-26 at 19 53 22

Latenight:
Screenshot 2021-12-26 at 19 52 54

Shade (tiny buttons):
Screenshot 2021-12-26 at 19 54 48

Tango (expected):
Screenshot 2021-12-26 at 19 55 40

**Comment by ronso0** [comment]
Okay, so this needs to be addressed by changing the stylesheets. I'll take a look.

This PR LGTM

**Review by ronso0 — APPROVED** [review]

**Comment by ronso0** [comment]
@ninomp Feel free to pick the stylesheet fix 49d10e59a1
(I'm too lazy to open a PR atm :)

The dialog now looks good (system style).

**Comment by ronso0** [comment]
@foss- Can you confirm the dialogs have the default system style now?

**Comment by foss-** [comment]
Confirming this is solved, dialog looks good in all skins.

**Comment by ronso0** [comment]
Great, thank you @ninomp for fixing and @foss- for testing :)

**Comment by ninomp** [comment]
@foss- Thank you for testing.
@ronso0 Thank you for assistance and review.

**Pull request merged**